geoutils/projtools.py

```diff
@@ -15,12 +15,9 @@
 from shapely.geometry.base import BaseGeometry
 from shapely.geometry.polygon import Polygon
 
-from geoutils.georaster import Raster
-from geoutils.geovector import Vector
-
 
 def bounds2poly(
-    boundsGeom: list[float] | rio.io.DatasetReader | Raster | Vector,
+    boundsGeom: list[float] | rio.io.DatasetReader,
```

Comment:
why did you delete it? It's actually meant to take a Raster or Vector instance.

Reply:
Cannot use `Raster` or `Vector` in `projtools` due to circular import